net-mail/automx2/automx2-2024.1.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{11,12} )
+
+inherit distutils-r1
+
+DESCRIPTION="Email client autoconfiguration service"
+HOMEPAGE="https://rseichter.github.io/automx2/"
+SRC_URI="https://github.com/rseichter/automx2/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="acct-user/automx2
+	dev-python/flask[${PYTHON_USEDEP}]
+	dev-python/flask-migrate[${PYTHON_USEDEP}]
+	dev-python/flask-sqlalchemy[${PYTHON_USEDEP}]
+	dev-python/ldap3[${PYTHON_USEDEP}]
+"
+
+distutils_enable_tests unittest
+
+python_prepare_all() {
+	sed -i -E -e 's/^(packages =).+/\1 automx2/' setup.cfg || die
+	distutils-r1_python_prepare_all
+}
```

Comment:
I was wondering why this package has to be patched here -- what's wrong with the upstream build system. Well, it looks like this modifies it to not package up `automx2.generators.*` and `automx2.views.*`, which aren't actually used by anything other than `automx2.server`. But I'm not sure I understand why they need to be removed or why that isn't, or why if they need to be removed (is it test-only code?) it isn't removed upstream as well.

Reply:
Now _that_ was useful information, unlike guessing games. My sole intention is to remove the offending `tests` folder before the ebuild's installation step, and I made a mistake. I will push an updated ebuild after writing this comment. I now use a proper patch for `setup.cfg`. As a useful byproduct the patch makes it easier for third parties to see the intended effect.